Src/AutoFakeItEasy/FakeItEasyRelay.cs

```diff
@@ -82,13 +82,28 @@ public object Create(object request, ISpecimenContext context)
 
             var fakeType = typeof(Fake<>).MakeGenericType(type);
 
-            var fake = context.Resolve(fakeType) as FakeItEasy.Configuration.IHideObjectMembers;
+            var fake = context.Resolve(fakeType);
             if (fake == null)

```

Comment:
I would expect the only change being made in this file is to replace `IHideObjectMembers` with a call to `IsFake` extension method (?)